code/game/gamemodes/roguetown/roguetown.dm

```diff
@@ -179,7 +179,7 @@ var/global/list/roguegamemodes = list("Rebellion", "Vampires and Werewolves", "N
 		//if(4)
 		//	log_game("Major Antagonist: None")
 
-	if(prob(80))
+	if(prob(100))
```

Comment:
This bit is somewhat redundant. if you want it to be 100% of the time, just remove the if statement.